lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -47,10 +47,10 @@ module Cardano.Wallet.Shelley.Compatibility
     , emptyGenesis
 
       -- * Eras
-    , AnyCardanoEra (..)
-    , AnyShelleyBasedEra (..)
-    , CardanoEra (..)
-    , ShelleyBasedEra (..)
+    , AnyCardanoEra(..)
+    , AnyShelleyBasedEra(..)
+    , CardanoEra(..)
+    , ShelleyBasedEra(..)
```

Comment:
```suggestion
    , AnyCardanoEra (..)
    , AnyShelleyBasedEra (..)
    , CardanoEra (..)
    , ShelleyBasedEra (..)
```

Autoformatter again?

Reply:
weird, I do not run autoformatters on old modules, they do a mess